README.md

```diff
@@ -26,10 +26,20 @@ The GOV.UK Prototype Kit is maintained by the Government Digital Service. If you
 * email [govuk-design-system-support@digital.cabinet-office.gov.uk](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) 
 * [get in touch on Slack](https://ukgovernmentdigital.slack.com/messages/prototype-kit)([open in app](slack://channel?team=T04V6EBTR&amp;id=C0647LW4R)) 
 * [view known issues on GitHub](https://github.com/alphagov/govuk-prototype-kit/issues)
-        
+
 ## Contributing
 
 If you’ve got an idea or suggestion you can:
 
 * [get in touch on the developer Slack channel](https://ukgovernmentdigital.slack.com/messages/prototype-kit-dev)([open in app](slack://channel?team=T04V6EBTR&amp;id=C0E1063DW))
 * [create a GitHub issue](https://github.com/alphagov/govuk-prototype-kit/issues)
+
+## Deploying to GOV.UK PaaS
+
+If you want to deploy your prototype to [GOV.UK PaaS](https://www.cloud.service.gov.uk) you can use the existing [application manifest](manifest.yml) and push using the following command:
+
+```
+cf push --var username=myusername --var password=secret --var prototype-name=my-new-service
```

Comment:
How can I make `secret` a genuinely secret password?

How can I use a password that's not visible in an open-source repo where I've configured my travis/concourse etc to run this command? How would I be storing my genuinely secret password?

Reply:
Depends on how you're doing your deployment. If you're using Travis (or similar) they usually have a secrets feature where you can replace `${some_secret}` with something you configure in their UI. For Travis: https://docs.travis-ci.com/user/environment-variables#defining-encrypted-variables-in-travisyml

If you're just pushing from your local machine you can use `--vars-file` instead of `--vars` and CF will read the secrets from a file (which you can then tell git to ignore).

Reply:
Thanks for that - it's super useful 😁 I don't know how frequent a use case this is for prototype kit users, but maybe good to add to the docs?

Reply:
You can use Travis or GitHub Actions secret:

When Travis runs
```
cf push --var username=hello --var "password=$PASSWORD" --var prototype-name=my-new-service
```

See Travis documentation: https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml

See GitHub Actions documentation: https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables

Reply:
I've attempted to incorporate this discussion and the suggestions posted by @bravegrape in the tutorial